guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -321,14 +321,23 @@ Let's start with the HTML we're working with:
 <button type="button">Pause</button>
 ```
 
-Next, let's add an event handler to the `Play` button:
+It might be tempting to try to handle this by managing the whole set of interactions at the component level. But there are three reasons we might want to tackle this using a modifier instead:
 
-```handlebars {data-filename="app/components/audio-player.hbs" data-diff="-3,+4"}
+1. Components don't have access to DOM elements directly. We'd have to render the page, push an element back up into the component class, and *then* wire it up. This would come with performance costs, because we would have to render twice before anything could work.
+2. We can't really use autotracking or 1-way data flow that way. If we wanted the caller of our component to be able to pass an argument in and have the component respond appropriately, we'd have to use something like `{{did-update}}`.
+3. None of the logic we built to manage playing and pausing would be reusable! It would be specific to just this component.
+
+Using a custom modifier, we can avoid all of these issues!
+
+First, let's add event handlers to the `Play` and `Pause` buttons:
```

Comment:
```suggestion
With these reasons in mind, let's look at how to create a modifier for our audio component. We will arrive at the final solution in incremental steps.

First, we add event handlers to the `Play` and `Pause` buttons:
```

Reply:
Took this idea and tweaked it a bit; see 414d25ee.